release/gcb/generate_manifest.sh

```diff
@@ -50,22 +50,27 @@ function checkout_code() {
 function create_manifest_check_consistency() {
   local MANIFEST_FILE=$1
 
- pushd istio
+  pushd istio
   local ISTIO_REPO_SHA
   local PROXY_REPO_SHA
+  local CNI_REPO_SHA
   local API_REPO_SHA
   ISTIO_REPO_SHA=$(git rev-parse HEAD)
+  pushd ../cni
+  CNI_REPO_SHA=$(git rev-parse HEAD)
```

Comment:
This should probably come from the istio.deps.

Reply:
One reason not to use istio.deps is that we have no separate release process for the CNI branch.   The only building and pushing of images is via this change set triggered by istio/istio triggers.   So therefore the CNI version would always be in lock-step with the istio/istio version.  I am not sure we are all on the same page here with what way this should be done.   Might need to discuss it directly.

Reply:
>> we have no separate release process for the CNI branch
I find that very similar to istio/api and istio/proxy. We just release them together with istio/istio.

I am proposing we can do the same to istio/cni, such that the dependent version is captured in istio.deps, and release process can just use it to do the right thing.

Reply:
Tim and I discussed I will add a CNI stanza to the istio.deps